pages/broker-assistance/[hackneyId]/assign-package.js

```diff
@@ -27,6 +29,18 @@ const breadcrumbs = [
   { text: 'Assign and attach a care plan' },
 ];
 
+const schema = yup.object().shape({
+  brokerId: yup.string().typeError('Please choose a Broker').required().min(2, 'Please choose a Broker'),
+  packageType: yup
+    .number()
+    .typeError('Please select a package type')
+    .required()
+    .min(1, 'Please select a package type'),
+  file: yup
+    .mixed()
+    .test('file', '', (value) => value?.size)
```

Comment:
we will want to test the file type here as well, i believe we only want to accept certain files. Check with martin/alex which file types